client/android-java/app/src/main/res/values/strings.xml

```diff
@@ -0,0 +1,4 @@
+<resources>
+    <string name="app_name">App</string>
```

Comment:
maybe set the App name to something more descriptive?

Reply:
I'm down to punt on this for now (alllll the samples are currently called app, I'd rather make one decision to keep or do a pass to rename them all).